samples/21.customization-plain-ui/README.md

```diff
@@ -0,0 +1,157 @@
+         <td>
+           <a href="https://reactjs.org/docs/higher-order-components.html">Higher-order component</a> to hoist specific features from Web Chat as props to the connected component. Although this is very similar to <a href="#connect-to-web-chat"><code>connectToWebChat</code></a>, it provides encapsulation of information and the business logic only required for the specific component type.
+         </td>
+      </tr>
+      <tr>
+         <td>
+            <a href="https://github.com/Microsoft/BotFramework-WebChat/blob/master/packages/component/src/connectToWebChat.js">
+               <a name="connect-to-web-chat"></a>
+               <code>connectToWebChat</code>
+            </a>
+         </td>
+         <td>
+           <a href="https://reactjs.org/docs/higher-order-components.html">Higher-order component</a> to expose all Web Chat UI API as props. The <code>connectToWebChat</code> method is very similar to Redux <a href="https://react-redux.js.org/api/connect"><code>connect</code></a> function, and provides access to all of the data required to build a chat component.
+         </td>
+      </tr>
+      <tr>
+         <td>
+            <a href="https://github.com/Microsoft/BotFramework-WebChat/blob/master/packages/component/src/Composer.js">
+               <code>Composer</code>
+            </a>
+         </td>
+         <td>
+            <p>React component which houses all components that requires access to Web Chat UI API.</p>
+            <p>This is very similar to Redux <code>&lt;Provider&gt;</code>.</p>
+         </td>
+      </tr>
+      <tr>
+         <td>
+            <p>
+               <a href="https://github.com/Microsoft/BotFramework-WebChat/tree/master/packages/core">
+                  <code>botframework-webchat-core</code>
+               </a>
+            </p>
+            <p>
+               <nobr>(Web Chat Core)</nobr>
+            </p>
+         </td>
+         <td></td>
+         <td>
+            <p>A stateful UI data store for a basic chat experience.</p>
+            <ul>
+               <li>Using Redux to provide a stateful data store but minimal support to various UI frameworks and app scenarios</li>
+               <li>UI framework agnostic: we think about building apps on React Native and Angular (using Redux bindings)</li>
+            </ul>
+         </td>
+      </tr>
+      <tr>
+         <td>
+            <a href="https://github.com/Microsoft/BotFramework-DirectLineJS">
+               <code>botframework-directlinejs</code>
+            </a>
+         </td>
+         <td></td>
+         <td>
+            <p>SDK to communicate between the browser and Direct Line channel service</p>
+            <ul>
+               <li>
+                  Designed to be event-driven (via observables) and stateless
+                  <ul>
+                     <li>Due to its stateless nature, for example, it does not understood suggested actions, because suggested actions depends on previous state</li>
```

Comment:
```suggestion
                     <li>Due to its stateless nature, for example, it does not understand suggested actions, because suggested actions depends on previous state</li>
```